Tests/TuistCoreTests/Models/XCFrameworkInfoPlistTests.swift

```diff
@@ -0,0 +1,15 @@
+import Basic
+import XCTest
+
+@testable import TuistCore
+@testable import TuistSupport
+@testable import TuistSupportTesting
+
+final class XCFrameworkInfoPlistTests: TuistUnitTestCase {
+    
+    func test_decode() {
+        let infoPlistPath = fixturePath(path: RelativePath("MyFramework.xcframework/Info.plist"))
+        let xcFrameworkInfoPlist: XCFrameworkInfoPlist = try! FileHandler.shared.readPlistFile(infoPlistPath)
+        XCTAssertNotNil(xcFrameworkInfoPlist)
+    }
```

Comment:
Seeing this method tests `FileHandler` perhaps best we move it to `FileHandlerTests`? (It doesn't have to test the XCFramework info.plist, it can be any plist file as the implementation in `FileHandler` is domain agnostic